src/test/java/com/sap/oss/phosphor/fosstars/data/artifact/VulnerabilitiesFromOwaspDependencyCheckTest.java

```diff
@@ -49,8 +50,9 @@ public void testVulnerabilitiesAvailable() throws IOException {
     Vulnerabilities vulnerabilities = values.of(VULNERABILITIES_IN_ARTIFACT).get().get();
     assertEquals(3, vulnerabilities.size());
 
-    Vulnerability vulnerability = vulnerabilities.entries().iterator().next();
-    assertEquals("CVE-2018-11307", vulnerability.id());
+    Set<String> vulnerabilityIds = new HashSet<>();
+    vulnerabilities.entries().forEach((vulnerability) -> vulnerabilityIds.add(vulnerability.id()));
+    assertTrue(vulnerabilityIds.contains("CVE-2018-11307"));
```

Comment:
I am thinking this can be shortened with help of stream(), as `vulnerabilities` is already a `Set`
```suggestion
    assertTrue(vulnerabilities.entries().stream()
        .anyMatch(vulnerability -> "CVE-2018-11307".equals(vulnerability.id())));
```